vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_w8a8_dynamictoken.py

```diff
@@ -6,13 +6,18 @@
 from vllm import _custom_ops as custom_ops
 from vllm.model_executor.layers.quantization.compressed_tensors.schemes import (
     CompressedTensorsScheme)
+from vllm.model_executor.layers.quantization.compressed_tensors.utils import (
```

Comment:
Do we need to do the same thing for static per tensor?

Reply:
Do we support channelwise for staticpertensor already?

Reply:
These changes only impact dynamic per token it seems, so I was jw

Reply:
Yup - just dynamic per token so far.  Easy addition to the static per tensor scheme - I can add it as part of this PR. Will be a few code lines to update.

Reply:
let's do as follow up